tests/test_formats.py

```diff
@@ -82,6 +90,9 @@ def test_date_format():
     t.set_format('time', ds.DateFormatter("%Y-%m-%d %H:%M:%S.%f"))
     assert isinstance(t['time'][0], float)
 
+    formatter = formats.DateFormatter()
```

Comment:
I think these two lines would be better in a separate test (a separate method)

Reply:
Done!